src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -1013,6 +1013,30 @@ public void run() {
                 CustomItemList.CoinWitchIV.get(1L),
                 100,
                 1920);
+        GT_Values.RA.addAssemblerRecipe(
+                CustomItemList.CoinFlower.get(1L),
+                GT_Utility.getIntegratedCircuit(10),
+                CustomItemList.CoinFlowerI.get(10L),
```

Comment:
Shouldn't this be other way around ?
```suggestion
                CustomItemList.CoinFlower.get(10L),
                GT_Utility.getIntegratedCircuit(10),
                CustomItemList.CoinFlowerI.get(1L),
```